backend/src/util/database.js

```diff
@@ -1,12 +1,17 @@
 let Sequelize = require("sequelize");
 
+const database = process.env.NODE_ENV === 'development'
+  ? process.env.DATABASE
+  : process.env.DATABASE_TEST;
+
```

Comment:
Determine which database to use on setup. You will need to add DATABASE_TEST to your .env